app.js

```diff
@@ -0,0 +1,68 @@
+const STORAGE_KEY = "tasks";
+ 
+function saveTasks(tasks) {
+  localStorage.setItem(STORAGE_KEY, JSON.stringify(tasks));
+}
+ 
+function loadTasks() {
+  const json = localStorage.getItem(STORAGE_KEY);
+  return json ? JSON.parse(json) : [];
+}
+ 
+function renderTasks() {
+  const tasks = loadTasks();
+  const board = document.getElementById("task-board");
+  board.innerHTML = "";
+ 
+  for (const task of tasks) {
+    const card = document.createElement("div");
+    card.className = "task-card";
+    card.innerHTML = `
+      <p>${task.text}</p>
+      <small>${task.date} <br> ${task.time}</small>
+      <span class="delete-btn" onclick="deleteTask(${task.id})">
+        <i class="bi bi-x-circle"></i>
+      </span>
+    `;
+    board.appendChild(card);
+    setTimeout(() => card.classList.add("show"), 10);
```

Comment:
you should have used css `@keyframes` to achieve this.
all in all animation gets wrong, it shows on any refresh, on any delete, on any note. should have appeared only once just when creating the note